src/main/java/io/appium/java_client/remote/MobileCapabilityType.java

```diff
@@ -31,11 +31,6 @@
      */
     String AUTOMATION_NAME = "automationName";
 
-    /**
-     * Which mobile OS platform to use.
-     */
-    String PLATFORM_NAME = "platformName";
```

Comment:
Where this constant is present now?

Reply:
in `org.openqa.selenium.remote.CapabilityType` 

```java
public interface MobileCapabilityType extends CapabilityType
```